Crew cannot be installed on x86_64 Chromebook since https://github.com/skycocker/chromebrew/pull/3828

## Description

The initial install (piping from Curl to Bash) and any later build terminates with the following failure...

```
checking for C compiler default output... configure: error: C compiler cannot create executables
ncurses failed to install: `../configure --prefix=/usr/local --libdir=/usr/local/lib64 --without-normal --with-shared --with-cxx-shared --without-debug --enable-pc-files --with-pkg-config-libdir=/usr/local/lib64/pkgconfig --enable-widec` exited with 77
```

This error first presented simply when running the install script on a powerwashed HP Chromebook 14 X360 14-da0000na. However, it can be recreated by trying to install anything at all, since the fundamental core packages didn't install correctly.

Output:
```shell
chronos@localhost / $ crew install rsync
rsync: rsync is an open source utility that provides fast incremental file transfer.
https://rsync.samba.org/
version 3.1.3
The following packages also need to be installed: 
ncurses gmp mpfr mpc isl cloog osl glibc gcc_tools linuxheaders make pkgconfig manpages gdbm readline groff uchardet libpipeline compressdoc doxygen sed acl attr gettext libxml2 zlibpkg icu4c automake autoconf m4 libsigsegv libtool intltool perl_xml_parser expat patch diffutils bison flex util_macros wget gnutls libunistring nettle openssl libressl libtasn1 trousers p11kit libffi libunbound libidn2 libpsl libmetalink gawk lzip unzip zip cmake meson ninja python3 bz2 xzutils sqlite libedit krb5 libtirpc re2c mandb 
Do you agree? [Y/n] y
Proceeding...
ncurses: The ncurses (new curses) library is a free software emulation of curses in System V Release 4.0 (SVr4), and more.
https://www.gnu.org/software/ncurses/
version 6.2
No precompiled binary available for your platform, downloading source...
################################################################################################################################################################### 100.0%
Archive downloaded
Unpacking archive using 'tar', this may take a while...
gcc (GCC) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
checking for egrep... grep -E
Configuring NCURSES 6.2 ABI 6 (Sun Feb 23 20:34:23 GMT 2020)
checking for package version... 6.2
checking for package patch date... 20200212
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
Configuring for linux-gnu
checking for prefix... /usr/local
checking for gnatgcc... no
checking for gcc... gcc
checking for C compiler default output... configure: error: C compiler cannot create executables
ncurses failed to install: `../configure --prefix=/usr/local --libdir=/usr/local/lib64 --without-normal --with-shared --with-cxx-shared --without-debug --enable-pc-files --with-pkg-config-libdir=/usr/local/lib64/pkgconfig --enable-widec` exited with 77
```

## Versions and system information
chromebrew: `crew --version` 

```
1.3.0
```

ChromeOS: `cat /etc/lsb-release`

```
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=6113849
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={495DCB07-E19A-4D7D-99B9-4710011A65B1}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={495DCB07-E19A-4D7D-99B9-4710011A65B1}
CHROMEOS_RELEASE_BOARD=nami-signed-mp-v5keys
CHROMEOS_RELEASE_BRANCH_NUMBER=81
CHROMEOS_RELEASE_BUILDER_PATH=nami-release/R79-12607.81.0
CHROMEOS_RELEASE_BUILD_NUMBER=12607
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=79
CHROMEOS_RELEASE_DESCRIPTION=12607.81.0 (Official Build) stable-channel nami 
CHROMEOS_RELEASE_KEYSET=mp-v5
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_UNIBUILD=1
CHROMEOS_RELEASE_VERSION=12607.81.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=12607.81.0
```

Model: HP Chromebook 14 X360 14-da0000na

Architecture: `uname -m`

```
x86_64
```

## Addtional information

At the time of writing the last commit to the Chromebrew repo ( https://github.com/skycocker/chromebrew/pull/3828 ) was suspiciously titled "Update ncurses from 6.1 to 6.2"

--- a/packages/ncurses.rb
+++ b/packages/ncurses.rb
@@ -8,8 +8,16 @@ class Ncurses < Package
   source_sha256 '30306e0c76e0f9f1f0de987cf1c82a5c21e1ce6568b9227f7da5b71cbea86c9d'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'e040d9c065046e107be2712904f0452720724254cc36aa7f04056246967b41b5',
+     armv7l: 'e040d9c065046e107be2712904f0452720724254cc36aa7f04056246967b41b5',
+       i686: '715c277f836a8f8f6c40736c667f63913d6e8ee005b4eb258a7a156a6b6db95a',
+     x86_64: '7ac9b423de77550b4ac2fc82ce4d42fc694352d3911918ba265ca2472d595f38',
   })
 
   def self.build

